main/lsp/requests/completion.cc

```diff
@@ -452,10 +523,13 @@ unique_ptr<ResponseMessage> LSPLoop::handleTextDocumentCompletion(LSPTypechecker
             return left.method._id < right.method._id;
         });
 
-        // TODO(jez) Do something smarter here than "all matching keywords always come first"
+        // TODO(jez) Do something smarter here than "all keywords then all locals then all methods"
```

Comment:
If you eventually score them by closeness of match then you could sort by score.

Reply:
Actually, this is what `sortText` is for, nevermind.

Reply:
Yeah the trick is that I'm not sure how to sort things relative to each other.